src/networkapi.jl

```diff
@@ -1481,5 +1481,158 @@ function validate(rs::ReactionSystem, info::String = "")
     validated
 end
 
+"""
+    iscomplexbalanced(rs::ReactionSystem, rates::Vector)
```

Comment:
```suggestion
    iscomplexbalanced(rs::ReactionSystem, rates)
```

Reply:
Also, probably it is more descriptive to call the second argument `parametermap` or similar as it should be a mapping giving numeric values for all system parameters.